### DISCUSSION TITLE
Raylib on replit?

### DISCUSSION THREAD

**Discussion by ProfAndreaPollini** [raysan5/raylib#2107] [Q&A]
Hi all,
  using replit in my classroom I found some examples of applications which use SDL (which is included in replit environment). I wonder how (if possibile) I can get also raylib compiled and running on replit. Anyone have tried it? I don't know how one can draw using SDL without using some tools like emscripten (which is not included in replit install).

Thanks a lot!

AP

**Comment by raysan5** [comment]
@ProfAndreaPollini If SDL can run then raylib can probably also run but I have no idea how. In any case I'll ask them.

**Comment by ghaberek** [comment]
I had this same idea today (exactly three years later!) so I threw together a Repl using [raylib-game-template](https://github.com/raysan5/raylib-game-template).

You can find the Repl template here: https://replit.com/@ghaberek/raylib-game-template?v=1

It actually seems very convenient to quickly bootstrap a game directly in the browser! 🤔

**Reply by demc** [reply]
Thanks! This is awesome.
